charts/velero/templates/deployment.yaml

```diff
@@ -4,6 +4,10 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "velero.fullname" . }}
+  annotations:
+    {{- with .Values.deploymentAnnotations }}
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
```

Comment:
```yaml
{{- if .Values.deploymentAnnotations }}
  annotations:
  {{- with .Values.deploymentAnnotations }}
    {{- toYaml . | nindent 4 }}
  {{- end }}
{{- end }}
```

Reply:
Thanks for the suggestion, I just applied it.

Reply:
I just realized that we could make it simpler.
```yaml
  {{- with .Values.deploymentAnnotations }}
  annotations:
    {{- toYaml . | nindent 4 }}
  {{- end }}
```